src/commands/infra/README.md

```diff
@@ -0,0 +1,11 @@
+## `spk infra`
```

Comment:
Let's update documentation on how to use `spk infra create`

Reply:
Updated!